### PR TITLE
Debug commands and render task capturing

### PR DESCRIPTION
**Pull request by kvark**
When we hook it up to Gecko, playing with those debug commands would allow us to quickly identify the failing area.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/2365)

### CODE REVIEW

File: webrender/src/renderer.rs [hunk 1]
```diff
@@ -2861,7 +2875,7 @@ impl Renderer {
 
             for &mut (_, RenderedDocument { ref mut frame, .. }) in &mut active_documents {
                 self.prepare_gpu_cache(frame);
-                assert!(frame.gpu_cache_frame_id <= self.gpu_cache_frame_id);
+                //assert!(frame.gpu_cache_frame_id <= self.gpu_cache_frame_id);
```
Comment (glennw) [inline webrender/src/renderer.rs]: I guess this is no longer valid? Should it be removed?

Comment (kvark) [reply webrender/src/renderer.rs]: Ah, you caught me!
Problem is that this assertion would false-fully trigger when replaying any capture made between the introduction of the assert and #2355. So I temporarily (!) commented it out when I was looking at a problem, and forgot about that when preparing the PR.

**Comment by kvark** [comment]
Should be fixed now :)

**Pull request merged**